spec/02-integration/08-status_api/01-core_routes_spec.lua

```diff
@@ -40,7 +40,31 @@ describe("Status API - with strategy #" .. strategy, function()
       assert.is_number(json.server.connections_writing)
       assert.is_number(json.server.connections_waiting)
       assert.is_number(json.server.total_requests)
+      assert.is_nil(json.server.configuration_hash)
     end)
+
+    it("/status returns status info with configuration_hash in dbless mode #off", function()
+      local res = assert(client:send {
+        method = "GET",
+        path = "/status"
+      })
+      local body = assert.res_status(200, res)
+      local json = cjson.decode(body)
+      assert.is_table(json.database)
+      assert.is_table(json.server)
+
+      assert.is_boolean(json.database.reachable)
+
+      assert.is_number(json.server.connections_accepted)
+      assert.is_number(json.server.connections_active)
+      assert.is_number(json.server.connections_handled)
+      assert.is_number(json.server.connections_reading)
+      assert.is_number(json.server.connections_writing)
+      assert.is_number(json.server.connections_waiting)
+      assert.is_number(json.server.total_requests)
+      assert.equal("no_configuration_yet", json.server.configuration_hash)
+    end)
```

Comment:
Missing testa: the value of the hash in the API is a hex (32 char long). This is returned once the data plane has a valid configuration.

Reply:
:+1: now included in 54f5a16e4d8c9b39581bd774a5241395f441f1d8